product_docs/docs/pgd/5/raft/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+
+title: Proxies, Raft and Raft subgroups
+
+---
+
+PGD manages its metadata using a  Raft model where a top-level group spans all the data nodes in the PGD cluster. A Raft leader is elected by the top-level group and propagates the state of the top-level group to all the other nodes in the group. 
+
+!!! Hint What is Raft?
+Raft is an industry accepted algorithm for making decisions though achieving “consensus” from a group of separate nodes within a distributed system. 
+!!!
+
+It is essential for certain operations in the cluster that the Raft leader – such as adding and removing nodes and allocating ranges for [galloc](sequences/#pdg-global-sequences) sequences – is both established and maintained. 
+
+It also means that one half of the nodes in the top-level group, plus one additional node must be able to reach each other. So, in a top-level group with five nodes, at least three of the nodes must be reachable by each other to establish a Raft leader.
```

Comment:
"one half plus one"? Isn't "absolute majority" a much simpler and well established description for exactly that? Or is that just my bias stemming from Swiss grassroot democracy, where that's a common term?

Reply:
"It also means that an absolute majority of nodes in the top-level group (one half of them plus one) must be able to reach each other. So, in a top-level group with five nodes, at least three of the nodes must be reachable by each other to establish a Raft leader." 

reworked to include both.... (absolute majority does get confused with unanimity by some folk)